java-shared-config/pom.xml

```diff
@@ -572,14 +572,9 @@
       </activation>
       <properties>
         <!-- default config values -->
-        <docletName>java-docfx-doclet-1.9.0</docletName>
-        <docletPath>${env.KOKORO_GFILE_DIR}/${docletName}.jar</docletPath>
         <outputpath>${project.build.directory}/docfx-yml</outputpath>
         <projectname>${project.artifactId}</projectname>
-        <excludeclasses></excludeclasses>
-        <excludePackages></excludePackages>
```

Comment:
Are there repos that depend on overwriting this value to get the behavior they want out of the plugin? Does removing it from additionalOptions thus break them?

Reply:
Yes, some of the handwritten repos have a custom `DocFx` profile, like java-bigtable: https://github.com/googleapis/java-bigtable/blob/main/pom.xml#L322

However, their profiles are configured to include any excluded classes and packages which should overwrite the entire default profile, so I don't think removing these additional options here will affect them.